### ISSUE TITLE
[Test Failure][Accessibility] “Dependencies” covered “Tags” in the description of the package on the right panel in PM UI

### ISSUE THREAD

**Issue by v-longmin** [NuGet/Home#7816]
## Details about Problem
VS Version: D16.0 28619.77
OS Version: 17763.RS5_Release.180914-1434.
## Detailed repro steps 
1. Create a new test certificate:.\CreateTestCertificate.ps1 –AddAsTrustedRootAuthority
2. Sign a package: nuget.exe sign <PackageFilepath> -CertificatePath <PfxFilePath>
3. Install the package into a project in PM UI, and observe the Description in the right panel.
## Expected:
“Tags” should not be covered by “Dependencies”.
## Actual:
“Dependencies” covered “Tags” as below screenshot.
![image](https://user-images.githubusercontent.com/40230425/53080659-1d709600-3534-11e9-9246-0d33c8369213.png)
## Notes
1. The issue does not repro on D16.0 28617.09 + NuGet Client Dev\5.0.0.5852.
2. The issue only repro on in the package I created, not in nuget.org.

**Comment by nkolev92** [comment]
@rrelyea We need to re-prioritize this. 

it seems like a regression with more than 1 repro.

**Comment by nkolev92** [comment]
There are 2 reported repros of this issue. 

- Empty dependencies element + tags element with values
- Empty dependencies element + tags element without values. 

Testing on the latest d16.0 Preview 4.2, I was not able to reproduce either one. 
The reported by Joel with an actual package given did not repro for me either. 

Looking through the code, the margin's seem right and it wasn't fully obvious what could be going wrong. 

@v-longmin Are you still able to repro this? If so, can you please provide the package in question?

**Comment by v-longmin** [comment]
@nkolev92 No, the issue does not repro on D16.1stg 28717.10 + NuGet Clinet Dev\5.1.0.5896.

**Comment by nkolev92** [comment]
While this does seem like something that needs fixing, I'm not sure I can do much without a repro on my end. Given that it's a very spotty repro I'd suggest moving to this to a future milestone. 

@rrelyea thoughts?

**Comment by nkolev92** [comment]
Synced up during stand-up. 

Moving this to a future release until we get a better repro.

**Comment by v-longmin** [comment]
The issue has fixed, closed it, thanks.
